sdk/rust/oak_config_serializer/src/main.rs

```diff
@@ -87,44 +87,67 @@ impl Default for InitialNodeConfig {
     }
 }
 
+const CACHE_DIRECTORY: &str = ".oak";
+
 /// Computes SHA256 sum from `data` and returns it as a HEX encoded string.
 fn get_sha256(data: &[u8]) -> String {
     let mut hasher = Sha256::new();
     hasher.update(data);
     hex::encode(hasher.finalize().as_slice().to_vec())
 }
 
-/// Load Wasm module from URL or a file, if URL is not specified.
+/// Download Wasm module from `url`.
```

Comment:
I see nothing about wasm modules in this fn, it seems a generic download file from URL?

Reply:
Changed